tests/test_input_doc.py

```diff
@@ -192,7 +227,7 @@ def _make_input_doc(path):
     in_doc = InputDocument(
         path_or_stream=path,
         format=InputFormat.PDF,
-        backend=PyPdfiumDocumentBackend,
+        backend=DoclingParseV4DocumentBackend,
```

Comment:
this is not producing GT test results
```suggestion
```

Reply:
The `InputDocument` __init__ has mandatory backend argument.

Reply:
Addressed.